src/sync/installation.ts

```diff
@@ -446,65 +439,62 @@ export async function maybeScheduleNextTask(
 const redis = new IORedis(getRedisInfo("installations-in-progress"));
 
 const RETRY_DELAY_BASE_SEC = 60;
-export const processInstallation =
```

Comment:
/nit
There're quite a bit of `git diff` in this section, do you think it is possible to just remove the `(app: Appliaction) => {` bit, but still keep the original format/indent? 

My personal preference will be try to have the `git diff` show as less as possible, but happy to follow other opinions.

Reply:
fair, have reverted the formatting... for now. you are right, its way easier to read without the new indent.

Will probably re-format it again after approval. The current indent just makes me feel too uncomfortable reading it lol